chef/cookbooks/nfs-server/metadata.json

```diff
@@ -6,7 +6,6 @@
     "replacing": {
     },
     "dependencies": {
-      "utils": []
```

Comment:
Could you quickly explain what we use the `dependencies` key for?

Reply:
This is the same as the usual `depends` on the metadata.rb of a chef recipe[0]. Supposedly, when you upload a cookbook to the chef server, it would compile the metadata.rb and store a json. If you only have the .json then it will not compile it and store the json directly after some checks.

I believe having only a .json is frowned upon, and the preference is to have the .rb file instead, but this is on more modern chef versions, not sure about chef 10.

[0] https://github.com/crowbar/crowbar-core/blob/master/chef/cookbooks/apache2/metadata.rb#L7

Reply:
Ah thanks @Itxaka! In usual Chef fashion, there is a multitude of ways of doing the same thing! :smile: Maybe this is due to Ruby's Perl heritage. :wink: